src/ProjectTemplates/test/Helpers/Project.cs

```diff
@@ -31,6 +31,8 @@ public class Project
 
         public SemaphoreSlim DotNetNewLock { get; set; }
         public SemaphoreSlim NodeLock { get; set; }
+        public string ProjectDirectory { get; set; }
+        public string AppName { get; set; }
```

Comment:
The hosted project is unique because newing it produces 3 subprojects. `AppName` is the name used during `dotnet new` which is then used to construct the names of the sub projects. This value is what ends up in the `title` tag in the blazor hosted project

Reply:
Can we avoid introudcing all these Blazor specific concepts into this class?

Reply:
I don't think there are other templates that produce more than one project file. If you can think of another way to represent this, I'm all ears